### PR TITLE
Added a list of some of the most relevant meetups

### PR DESCRIPTION
**Pull request by MyPitit**

### CODE REVIEW

File: networking.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+
+
+##### [London Emberjs User Group](http://www.meetup.com/London-Emberjs-User-Group/)
+
+ Monthly meetups with talks from contributors, library authors and enthusiasts. The aim is to cover a broad range of topics of interest to beginners and advanced users alike.
+
+
+##### [London react user group](http://www.meetup.com/London-React-User-Group/)
+
+Meetups once a month, usually the 3rd Wednesday of each month. Generally have 2 or 3 speakers and loads of questions, pizzas and beer.
```
Comment (iteles) [inline networking.md]: Would add a note here that the tickets fly out of the door, so a reminder on your phone for when they become available is always useful :+1:

File: networking.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+##### [HTML5 CodeShow](http://www.meetup.com/HTML5-CodeShow/)
+
+A group of hackers/developers/tech enthusiasts who are ecstatic about what the future of the web and mobile apps will bring.
+
+## FrontEnd
+##### [Endgineers London](http://www.meetup.com/Front-Endgineers-London/)
+The Front Endgineers is a group for Software Engineers with an interest in Front End technologies.
+
+## General
+##### [Women who code](http://www.meetup.com/Women-Who-Code-London/)
```
Comment (iteles) [inline networking.md]: Can we add Ladies of Code here as well (formerly Ladies Who Code)? http://www.meetup.com/Ladies-of-Code-UK/
Women who code overtook us in November last year around the same time I eased off the organising of the meetup (:sob:) but still pretty big :wink:

Comment (MyPitit) [inline networking.md]: Sure @iteles I will add this meetup :)

File: networking.md (hunk 1 above)
Comment (MyPitit) [inline networking.md]: Yes you are right I will add a note.

**Pull request merged**